zrml/prediction-markets/src/migrations.rs

```diff
@@ -516,15 +516,16 @@ mod tests {
         pool_is_closed: bool,
     ) {
         let amount = 100 * BASE;
+        let category_count = 5;
         assert_ok!(PredictionMarkets::create_cpmm_market_and_deploy_assets(
             Origin::signed(ALICE),
             BOB,
             period,
             gen_metadata(0),
-            MarketType::Categorical(5),
+            MarketType::Categorical(category_count),
             MarketDisputeMechanism::Authorized(CHARLIE),
             amount,
-            vec![BASE; 6],
+            vec![BASE; category_count.into()],
```

Comment:
We should not change existing migrations, since when the storage migration is executed, it checks if the storage version equals to a version where those values were still valid. I think you know that already, the PR is just a bit older.
However, since https://github.com/zeitgeistpm/zeitgeist/pull/680 removes all the code, a merge from main will automatically fix this.

Reply:
Fixed in b4d77450266e713a67a89cde0807bfe4fb46ad34

Reply:
There are some exceptions though, for example migrations that adjust the storage based on the value of a configuration value. Such migrations are thought to be used anytime a configuration value changes that for example affects the contents of the storage, thus they should be maintained ([see this example](https://github.com/PureStake/moonbeam/pull/1096))